cdap-security/src/test/java/co/cask/cdap/security/impersonation/UGIProviderTest.java

```diff
@@ -1,238 +1,247 @@
-/*
```

Comment:
Please ignore for now. Working on updating this test to be in line with the changes.